src/sql/workbench/services/connection/test/browser/connectionManagementService.test.ts

```diff
@@ -732,6 +732,18 @@ suite('SQL ConnectionManagementService tests', () => {
 		}
 	});
 
+	test('getProviderNames should return all known providers', done => {
+		try {
+			let providerNames = connectionManagementService.getProviderNames();
+			assert.equal(providerNames.length, 2);
+			assert.equal(providerNames.indexOf('MSSQL') > -1, true);
+			assert.equal(providerNames.indexOf('PGSQL') > -1, true);
+			done();
+		} catch (error) {
+			done(error);
+		}
```

Comment:
Make this method async and then you can just remove the try/catch - the error will fail the test in that case. You can also remove the done call as well since that won't be necessary (it'll pass if it completes the method without erroring)

Reply:
It is not really doing any async calls so just removed done() and try catch.